lib/class-wp-rest-block-editor-settings-controller.php

```diff
@@ -75,7 +75,7 @@ public function get_items_permissions_check( $request ) {// phpcs:ignore Variabl
 	 * @return WP_Error|WP_REST_Response Response object on success, or WP_Error object on failure.
 	 */
 	public function get_items( $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis
-		$editor_context = new WP_Block_Editor_Context();
+		$editor_context = new WP_Block_Editor_Context( array( 'name' => 'core/edit-site' ) );
```

Comment:
This one is tricky. Technically there's no screen at all since it's just a REST API endpoint. I did some digging and this endpoint is used by the mobile apps for Global Styles (https://github.com/WordPress/gutenberg/issues/29063), so I think using `core/edit-site` makes sense.

Reply:
Is this the same thing as the 'site' entity?
https://github.com/WordPress/gutenberg/blob/5f265e9e4200479f50d7310a40cc6c8a755fedb8/packages/core-data/src/entities.js#L28-L36

Reply:
Feels like it probably shouldn't have a `name`, or the `name` should be passed as a param to the REST request.

Reply:
> Is this the same thing as the 'site' entity?

It's not, no. I can't see anywhere where this endpoint is being used but I assume from the issue (https://github.com/WordPress/gutenberg/issues/29063) that it's used by the mobile app somehow.

Reply:
> Feels like it probably shouldn't have a `name`

Yeah maybe name needs to be optional after all 😕

Reply:
@geriux worked on the mobile integration so he might be the best person to share recommendations here.

Reply:
> I can't see anywhere where this endpoint is being used but I assume from the issue (https://github.com/WordPress/gutenberg/issues/29063) that it's used by the mobile app somehow.

That's correct, this is currently used on the WordPress mobile app to load the global styles data.

Reply:
In regards to the name, the one that's currently set `core/edit-site` doesn't affect the response data we need so that works.

Reply:
Thanks for confirming @geriux.

I looked at https://github.com/WordPress/gutenberg/pull/29969 more closely and realised that this endpoint already has a `'context'` param which we can use to set `$context->name` correctly! Done in e739e41.